lib/upgrade_format_task.rb

```diff
@@ -29,28 +29,28 @@ def upgrade(store, name, options = {})
       descriptions = [name]
     end
 
-    migrations_done = 0
+    errors = []
+
     descriptions.each do |description|
       begin
         Machinery.logger.info "Upgrading description \"#{description}\""
-        migrated = Migration.migrate_description(store, description, force: options[:force])
-        migrations_done += 1 if migrated
+        Migration.migrate_description(store, description, force: options[:force])
```

Comment:
If you don't check the return value anymore you miss the case where the description doesn't need to be upgraded. Then you get the message that the description is up to date as well as that the description is upgraded.